cmd/receptor.go

```diff
@@ -23,22 +23,33 @@ type nodeCfg struct {
 	DataDir      string `description:"Directory in which to store node data"`
 }
 
-func (cfg nodeCfg) Prepare() error {
+func (cfg nodeCfg) Init() error {
 	var err error
 	if cfg.ID == "" {
-		cfg.ID, err = os.Hostname()
+		host, err := os.Hostname()
 		if err != nil {
 			return err
 		}
+		lchost := strings.ToLower(host)
+		if lchost == "localhost" || lchost[:10] == "localhost." {
```

Comment:
should we use https://golang.org/pkg/strings/#HasPrefix here?

Reply:
Yeah, that's probably better. I'll update it.